scripts/run_interop_tests.sh

```diff
@@ -47,6 +47,14 @@ docker rm -f "$pid1"
 docker rm -f "$pid2"
 
 
+#
+# Run interop tests against grpc-web java connector code
+#
+docker-compose build java-interop-server
```

Comment:
Nit: can we merge this line with L34 above? So we build all the docker images first.

Reply:
done